docs/efiling/efiling_through_docassemble.md

```diff
@@ -0,0 +1,190 @@
+let you add capabilities and features to your interview, it's also complex, and some of that complexity will need
+to be embedded into your interview.
+
+## Before starting
+
+Before starting this guide, you should have:
+
+* a working docassemble interview, preferably using the AssemblyLine.
+* If you don't have one yet, follow [the steps to author your interview](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/planning).
+* contacted us for access to the EFSP server. We aren't yet making access publicly available, but will include an interest form here when we do.
+
+## Steps
+
+To start, set your docassemble server's [configuration](https://docassemble.org/docs/config.html). You only need to do this step once per server.
+
+In the configuration file, add the below block, replacing each value on the right with the information that we provide to you.
+
+```yaml
+efile proxy:
+  api key: abcDEFghiJKL123456789
+  url: https://efile.serverurl.com:9000
+  jeffnet api token: ABCDEF123
+  global waiver: 123a4567-b89c-1234-abcd-123456789000
+  global server admins:
+    - your_email@example.com
+```
+
+### Including the EFSP Library
+
+Next, add our docassemble EFSP library to your interview.
+
+```yaml
+---
+include:
+  - docassemble.EFSPIntegration:efiling_integration.yml
+  - docassemble.EFSPIntegration:login_qs.yml
+---
+```
+
+Next, you can create a client connection to our EFSP server.
+
+```yaml
+code: |
+  proxy_conn = ProxyConnection(credentials_code_block='tyler_login', default_jurisdiction='illinois')
+```
+
+The `credentials_code_block` is a YAML code block that handles the screens shown to the user to login to the Tyler system.
+We provide a code block, one that sets `tyler_login` at the end, and shows the appropriate screens for logging in (also handling
+password resets and incorrect logins). You can define your own code block as well, as long as if sets the trigger variable to
+`True` only if `proxy_conn.authenticate_user(...).is_ok().`.
+
+The `default_jurisdiction` is the jurisdiction that you are filing in. Currently, we only support `illinois`, `louisiana`, and `massachusetts`.
+
+### Get the information that you need to file the doc
+
+We're going to start with the simpler to implement situation: when some one is starting a specific type of a new case with the court, aka an initial filing.
+
+Since we're only filing one type of case, we can hardcode a few pieces of information in our interview:
+
+* the `tyler_case_category`
+* the `tyler_case_type`
+* the `case_subtype` (if necessary)
+* individual's `party_type`
+* several for the filing itself:
+  * `document_type`
+  * `filing_component`
+  * `tyler_filing_type`
+  * `filing_parties`
+  * `filing_action`
+
+To find the right codes, you can navigate through the URLs provided by visiting <https://efile.suffolklitlab.org:9000/jurisdictions/illinois/codes>.
```

Comment:
These links don't display anything for me in Firefox--looks like just an empty page.

Reply:
You mean that page itself? That's expected, I haven't deployed the latest proxy to prod yet, but I'll do that today and double check all of the links.

Reply:
Tyler seems to have buggered up their TLS certs during a migration, and I can't deploy a new version when their site is broken, so this'll have to just be wrong for a bit.